### DISCUSSION TITLE
Can you change the go get command

### DISCUSSION THREAD

**Discussion by KevinM2k** [renovatebot/renovate#23244] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.14.4

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Ability to change the `go get -d -t ./...` path to remove the flags or potentially change them to `go get -u ./...`

### What you tried so far.

Not found anything that can do this yet.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You cannot. Can you describe why the current ones don't work for you and why the change would?
